include/zephyr/logging/log_backend_rpmsg.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2025 Lexmark
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_LOG_BACKEND_RPMSG_H__
```

Comment:
Would you keep a single underscore at the end to meet common code style?

Reply:
none of the checks tagged this...will fix.